.dockerignore

```diff
@@ -1,21 +1,20 @@
-# Ignore all subdirectories
-*/*
+# Ignore everything
+*
 
 # Allow to run the test script inside the Docker container
 !/docker/test_dockerimage.sh
 
-# Ignore unnecessary files inside top-level directory
-*.bat
-*.csh
-*.fish
-*.ps1
-*.pyc
-.emscripten
-.emscripten.old
-.emscripten_cache
-.emscripten_cache__last_clear
-.emscripten_sanity
-.emscripten_sanity_wasm
-.flake8
-emscripten-releases-tot.txt
-README.md
+# Allow license file
+!LICENSE
+
+# Allow necessary build files in top-level directory
+!emscripten-releases-tags.json
+!emsdk
+!emsdk.py
+!emsdk_env.sh
+!emsdk_manifest.json
+
+# TODO: Somehow these legacy files are also required for building
```

Comment:
The files themselves are not legacy.. they just include the information for installation of legacy SDK.. which is still apparently allowed.

Reply:
Ah, I assumed they were not needed. Commit a1edad8c7b0eb996c1ee7a2d39b4403907343e8f fixes the description.